### DISCUSSION TITLE
Marker not rendering in Nextjs project using react-google-maps-api

### DISCUSSION THREAD

**Discussion by AbhayaShankar** [JustFly1984/react-google-maps-api#3304] [General]
While using `GoogleMap` and `Marker` component in `"use client"` in NextJS 13, 
Marker Component is not rendering at all.

```tsx
function Map() {
  return (
    <GoogleMap
      zoom={10}
      center={{ lat: 28, lng: 82}}
      mapContainerClassName="h-[30rem] w-full"
    >
      <Marker position={{ lat: 28, lng: 82}} />
    </GoogleMap>
  );
}
```

What could be causing this? And how to resolve this bug ?

**Comment by penguinuux** [comment]
Hello @AbhayaShankar!

Can you provide more context regarding your issue? Do you receive any errors in the console? Have you tried running the project using the 'useJsApiLoader' launcher?
